synapse/models/orgs.py

```diff
@@ -38,6 +38,7 @@ def getBaseModels():
                  {'subof': 'comp', 'fields': 'org,ou:org|sub,ou:org', 'doc': 'An org which owns a sub org'}),
                 ('ou:member', {'subof': 'comp', 'fields': 'org,ou:org|person,ps:person',
                                'doc': 'A person who is (or was) a member of an organization.'}),
+                ('ou:hasalias', {'subof': 'comp', 'fields': 'org,ou:org|alias,ou:alias'}),
```

Comment:
Technically this is a deprecated syntax ( the newer stuff uses name=type, ) but it's not a show stopper.

Reply:
changing

Reply:
I'll just change them all of ou:has*